ini/example_ini/cross_platform.ini

```diff
@@ -1,7 +1,7 @@
 [general_settings]
 sim_type = yue
 holding_time = 0.2
-arrival_rate = {'start': 100, 'stop': 102, 'step': 50}
+erlangs = {'start': 100, 'stop': 102, 'step': 50}
```

Comment:
I think this is a reason one of the pipelines are failing, please change the sim type to "arash" and it should work for now until we change it again :melting_face: